docs/pages/database-access/guides/mongodb-self-hosted.mdx

```diff
@@ -20,6 +20,15 @@ In this guide you will:
 
 ## Prerequisites
 
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+<Notice type="warning">
```

Comment:
This isn't doing any harm, and this is a backport, so doing nothing is a perfectly acceptable response.

That said, Teleport 7 is long EOL, so this warning isn't strictly necessary at this point.

Same goes for the other mentions of Teleport 7 (and even 6.2!) below.

Reply:
Handling in https://github.com/gravitational/teleport/pull/21539